src/Tools/Utils.php

```diff
@@ -5,6 +5,7 @@
 use Illuminate\Routing\Route;
 use League\Flysystem\Filesystem;
 use League\Flysystem\Adapter\Local;
+use Symfony\Component\VarExporter\VarExporter;
```

Comment:
I'm not sure I agree with the StyleCI rules around import ordering, but I will follow the rules :)

Reply:
You're not alone in that. I think it's a stupid rule.